src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -31,6 +31,16 @@
     private static final String CURRENT_DIRECTORY = System.getProperty("user.dir");
     public static final String gatkDirectory = System.getProperty("gatkdir", CURRENT_DIRECTORY) + "/";
 
+    // shortname of the project that stores the data and will run the code.
+    public final static String TEST_PROJECT = System.getenv("HELLBENDER_TEST_PROJECT");
+    // API key for TEST_PROJECT
+    public final static String TEST_APIKEY = System.getenv("HELLBENDER_TEST_APIKEY");
+    // A writeable folder on the project's GCS
+    public final static String TEST_STAGING = System.getenv("HELLBENDER_TEST_STAGING");
+    // A GCS path (on that project) where the test inputs are stored.
+    public final static String TEST_INPUTS = System.getenv("HELLBENDER_TEST_INPUTS");
```

Comment:
Now that these are in `BaseTest` can you change the prefixes on these constants from `TEST_` to `DATAFLOW_TEST_` to make it clear that they're for use with dataflow tests? Eg., `TEST_PROJECT` -> `DATAFLOW_TEST_PROJECT`. Can keep the names of the environment variables themselves the same.